model/project_event.go

```diff
@@ -0,0 +1,132 @@
+package model
+
+import (
+	"reflect"
+	"time"
+
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/evergreen-ci/evergreen/model/event"
+	"github.com/mongodb/grip/message"
+	"github.com/pkg/errors"
+	"gopkg.in/mgo.v2/bson"
+)
+
+const (
+	ResourceTypeProject      = "PROJECT"
+	EventTypeProjectModified = "PROJECT_MODIFIED"
+	EventTypeProjectAdded    = "PROJECT_ADDED"
+)
+
+type ProjectSettings struct {
+	ProjectRef         ProjectRef           `bson:"proj_ref" json:"proj_ref"`
+	GitHubHooksEnabled bool                 `bson:"github_hooks_enabled" json:"github_hooks_enabled"`
+	Vars               ProjectVars          `bson:"vars" json:"vars"`
+	Aliases            []ProjectAlias       `bson:"aliases" json:"aliases"`
+	Subscriptions      []event.Subscription `bson:"subscriptions" json:"subscriptions"`
+}
+
+type ProjectChange struct {
+	User   string          `bson:"user" json:"user"`
+	Before ProjectSettings `bson:"before" json:"before"`
+	After  ProjectSettings `bson:"after" json:"after"`
+}
+
+// Embed the EventLogEntry so we can reimplement the SetBSON method for Projects
```

Comment:
this comment doesn't 100% make sense to me. 

also can the word event be included in all of the public types/constants in this file.